src/core/space_interaction_API.jl

```diff
@@ -380,69 +377,13 @@ is expensive since in this case the allocating version can be more performant.
 function random_nearby_position(pos, model, r=1, f = nothing, alloc = false; kwargs...)
     iter = nearby_positions(pos, model, r; kwargs...)
     if isnothing(f)
-        return resorvoir_sampling_single(iter, model)
+        return itsample(abmrng(model), iter)
```

Comment:
Does `itsample` come from your package? I would say, preface the functions that come from your package with `IteratorSampling.function_name`.

Reply:
yes, good idea